app/views/teachers/TeacherStudentView.coffee

```diff
@@ -228,37 +228,48 @@ module.exports = class TeacherStudentView extends RootView
     for versionedCourse in @classroom.getSortedCourses() or []
       course = @courses.get(versionedCourse._id)
       numbers = []
+      performanceNumbers = []
       studentCourseTotal = 0
+      performanceStudentCourseTotal = 0
       members = 0 #this is the COUNT for our standard deviation, number of members who have played all of the levels this student has played.
       for member in @classroom.get('members')
         number = 0
+        performanceNumber = 0
         memberPlayed = 0 # number of levels a member has played that this student has also played
         for versionedLevel in versionedCourse.levels
           sessions = (levelSessionsByStudentByLevel[member] ? {})[versionedLevel.original] ? []
           for session in sessions
             playedLevel = levelDataByLevel[session.get('level').original]
             if playedLevel.levelProgress is 'complete' or playedLevel.levelProgress is 'started'
               number += session.get('playtime') or 0
+              performanceNumber += !playedLevel.isLadder and !playedLevel.isProject and session.get('playtime') or 0
```

Comment:
I think `playedLevel` here might not have a `isLadder` or `isProject` attribute set? Usually those would be `Level` [model methods](https://github.com/codecombat/codecombat/blob/5ee08dd2a78463e2b3f8c6c37494c8e5c145eb77/app/models/Level.coffee#L287-L291), and the underlying object used would have to have `type` and `shareable` projected/loaded into it. I think this [doesn't have either](https://github.com/codecombat/codecombat/blob/5ee08dd2a78463e2b3f8c6c37494c8e5c145eb77/app/views/teachers/TeacherStudentView.coffee#L466-L476), unless I'm missing something (haven't run it). Don't think the `shareable` is [included in the loaded data yet](https://github.com/codecombat/codecombat/blob/5ee08dd2a78463e2b3f8c6c37494c8e5c145eb77/app/views/teachers/TeacherStudentView.coffee#L79-L79).

Prefer `not` to `!` in CoffeeScript code.

Reply:
Changed `!` to `not`.

The `isLadder` and `isProject` properties are added here:
https://github.com/codecombat/codecombat/pull/6587/files#diff-078d9682cf26cae1a71e995f010706cf1ce04fd8e32e05f9b2ff573b61406581R489-R490